server/data/orchestrationApiClient.ts

```diff
@@ -139,18 +139,21 @@ export default class OrchestrationApiClient {
     prisonerId,
     visitorIds,
     excludedApplicationReference,
+    username,
   }: {
     prisonId: string
     prisonerId: string
     visitorIds: number[]
     excludedApplicationReference?: string
+    username: string
   }): Promise<AvailableVisitSessionDto[]> {
     return this.restClient.get({
       path: '/visit-sessions/available',
       query: new URLSearchParams({
         prisonId,
         prisonerId,
         visitors: visitorIds.join(','),
+        username,
```

Comment:
```suggestion
    bookerReference,
  }: {
    prisonId: string
    prisonerId: string
    visitorIds: number[]
    excludedApplicationReference?: string
    bookerReference: string
  }): Promise<AvailableVisitSessionDto[]> {
    return this.restClient.get({
      path: '/visit-sessions/available',
      query: new URLSearchParams({
        prisonId,
        prisonerId,
        visitors: visitorIds.join(','),
        username: bookerReference,
```

Personal preference I guess, but it might make more sense within our application to use `bookerReference` rather than `username` when we're passing it around here and in other files  - because that's what it actually is. Then right at the point of sending to the API we map it to `username`.